shell/platform/embedder/tests/embedder_frozen.h

```diff
@@ -0,0 +1,69 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef FLUTTER_SHELL_PLATFORM_EMBEDDER_EMBEDDER_FROZEN_H_
+#define FLUTTER_SHELL_PLATFORM_EMBEDDER_EMBEDDER_FROZEN_H_
+
+#include "flutter/shell/platform/embedder/embedder.h"
+
+// This file contains the subset of the embedder API that is "frozen".
+// "Frozen" APIs must not change to guarantee forward and backward ABI
+// compatibility.
+//
+// These changes are not allowed:
+// - New struct members must not be added.
+//
+// Structs in this file are copies of structs in the embedder API.
+// This allows tests to detect changes to frozen APIs.
+
+namespace flutter {
+
```

Comment:
This can be `namespace testing {` within the `flutter` namespace since this is specifically test code.

Reply:
Oops, missed that. Fixed!